packages/server/src/main/resources/application-devES.properties

```diff
@@ -0,0 +1,8 @@
+
+########################################################################################################################
```

Comment:
should this be in the base application.properties? And how are you going to handle the local dev version where project search will be local ES and dkg will be pointing to dev?

Reply:
I'd put it here so the default application.properties wouldn't be affected for local dev, but, you're right that it won't handle local project searching at all like this...